lib/run-compilation.js

```diff
@@ -14,6 +14,7 @@ var errorMessage = require('./constants').errorMessage;
 module.exports = function runCompilation (options, compiler, done) {
   var errors = [];
   var warnings = [];
+  var compilerMethod = options.emitErrors ? 'after-compile' : 'after-emit';
```

Comment:
Sure, I can explore writing some coverage for this (your tests cover a significant portion of this already though, but I can do some integration tests to validate the output of webpack/etc => otherwise I don't see much value in writing unit-tests to support this change). On a side note, I did have some cross-environment issues with spinning up your main script (npm test), so I implemented [cross-env](https://github.com/kentcdodds/cross-env) for setting run-time node variables across various platforms. I can also put a separate PR in to support implementing cross-env if that works for you?